app/jobs/bulk_affiliate_styles_uploader_job.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class BulkAffiliateStylesUploaderJob < ApplicationJob
+  queue_as :searchgov
+
+  def perform(user, filename, filepath)
+    @user = user
+    @filename = filename
+    @uploader = BulkAffiliateStylesUploader.new(filename, filepath)
+    @uploader.upload
+    report_results
+  end
+
+  def report_results
```

Comment:
If the parent method and/or children methods of this class  doesn’t have to be publicly accessed by the rest of the app, let’s make them private.